redaxo/src/addons/mediapool/functions/function_rex_mediapool.php

```diff
@@ -94,9 +94,34 @@ function rex_mediapool_saveMedia($FILE, $rex_file_category, $FILEINFOS, $userlog
     if ($isFileUpload) { // Fileupload?
```

Comment:
Ich habe doch auch nochmal eine grundsätzliche Frage: Aktuell ist der EP nur in diesem Zweig drin, also wenn es ein wirklicher Upload ist, nicht wenn irgendein Addon oder so diese Funktion aufruft.
Überlege, ob der EP nicht doch eher da auch greifen sollte, und wir dann eventuell doch einen neuen Namen brauchen?
Also will man als Addon den anderen Fall nicht genauso prüfen?

Reply:
Naja ich denke das hängt davon ab, ob ihr die Medienpool-Funktionen als öffentliche API anseht. Ich bin jetzt von der Dokumentation auf redaxo.org ausgegangen, wo der Medienpool in dem Kontext nicht auftaucht. Deshalb habe ich die Funktion als intern angenommen und damit als "wird nur vom Medienpool aufgerufen". Wenn ihr sagt, dass die Funktion eigentlich schon öffentlich ist, würde ich den EP tatsächlich auch in den `else`-Zweig übernehmen und ggf. in `MEDIA_PRE_SAVE` umbennen, wie es auch schonmal vorgeschlagen wurde.

Reply:
Die Funktion ist durchaus eine öffentliche Api (auch wenn für R5 unüblich als klassische Funktion).
Also gerne so ändern, dass es auch für den else-Zweig greift. 

Bzgl. Name: Wir haben zum Beispiel auch bereits `SLICE_ADD` und `SLICE_ADDED` etc.. Analog könnte man es hier auch nur `MEDIA_SAVE` nennen, und falls man irgendwann einen nach dem Speichern ergänzt, würde der dann `MEDIA_SAVED` heißen.

Reply:
Na aktuell gibt es doch schon einen `MEDIA_ADDED` Extension Point. Dann vielleicht lieber `MEDIA_ADD`? Ich ändere jetzt erstmal nur die Platzierung des EP, den Namen würde ich dann in einem abschließenden Commit setzten.

Reply:
Stimmt, bin dann für `MEDIA_ADD` 👍